pkg/kubelet/metrics/metrics.go

```diff
@@ -530,6 +555,10 @@ func Register(collectors ...metrics.StableCollector) {
 		legacyregistry.MustRegister(StartedPodsErrorsTotal)
 		legacyregistry.MustRegister(StartedContainersTotal)
 		legacyregistry.MustRegister(StartedContainersErrorsTotal)
+		if runtime.GOOS == "windows" {
```

Comment:
That suggests to me that maybe we're doing the wrong thing... it seems unlikely to me that this is the first-ever windows only metric. Do we have an example of one we can look at?

Reply:
I actually can't find any metrics in kubernetes/kubernetes components that are only applicable to Windows.
@claudiubelu or @jsturtevant are you aware of any?

A lot of the Windows specific differences are to adjust for behavioral differences on Windows vs. Linux (setting file permissions, adjusting file paths, querying system resources, etc) and any relative information exposed by stats/metrics are reported to users the same way.

There are a small number of metrics that are not applicable to Windows (`cgroup_manager_duration_seconds` and `evictions`  for example) that are always registered.

From a UX perspective would it be better to register these metrics for all OS's and note that they are Windows-specific or only register them on Windows?

My thoughts were that many Windows users are familiar with Linux concepts but many Linux users are not familiar with Windows concepts so we could just hide the Windows specific ones from users.
I don't have a very strong opinion here tho 😀

Reply:
I don't think we have any Windows specific metrics, this would be the first. I think there is opportunity for windows specific metrics but none are implemented now.

Reply:
They won't need to be explicitly hidden for Linux users since there won't be any Windows metrics emitted for a Linux machine.

Reply:
Sounds good to me. I removed the GOOS check conditional in the metrics registration logic.